### ISSUE TITLE
Seems that the meanIOU is not correct

### ISSUE THREAD

**Issue by liyin2015** [martinkersner/train-CRF-RNN#32]
I used the pretrained model from the official crfasrnn caffemodel, and used the test_model.py to test the the meanIOU on the test dataset of PASCAL VOC 2012 validation set, the meanIOU is only 0.400, which seems wrong. So, is there any problem with the evaluation metrics in the code?
